save-common/src/commonMain/kotlin/com/saveourtool/save/core/result/DebugInfo.kt

```diff
@@ -35,4 +35,18 @@ data class CountWarnings(
     val matched: Int,
     val expected: Int,
     val unexpected: Int,
-)
+) {
+    companion object {
+        const val NOT_APPLICABLE_COUNTER: Int = -99
```

Comment:
Maybe also add method to check if DebugInfo or a particular counter is `not applicable`? To fully encapsulate this logic here and only call this method from sae-coud

Reply:
good point, will do

Reply:
done